new-client/src/plugins/collector/components/Page.js

```diff
@@ -573,8 +573,47 @@ class Page extends Component {
       </Button>
     );
 
+    const collectAgainButton = (
+      <Button
+        variant="outlined"
+        color="primary"
+        className={classes.buttonRight}
+        onClick={() => {
+          this.props.model.observer.publish("abort");
+        }}
+      >
+        Tyck till igen!
+      </Button>
+    );
```

Comment:
`collectAgainButton` and `okButton` are virtually the same, except for the label.

Just use one of them, read the value of `this.props.options.collectAgain` inside it and set label accordingly.